tests/test_generation/exhaustive/__snapshots__/test_exhaustive/test_sync[models.py].raw

```diff
@@ -66,17 +66,27 @@ _implicit_subclass_packages: Set[str] = {
     'fastapi',
 }
 
+def _module_to_parent_package(frame):
+    try:
+        module_name = inspect.getmodule(frame).__name__
+        parent_package, *_ = module_name.split(".")
+        return parent_package
+    except:
+        return None
 
-def _maybe_warn_subclassing(new_model: str, base_model: str, *, stacklevel: int = 3) -> None:
+def _maybe_warn_subclassing(new_model: str, base_model: str, *, max_call_stack_depth_to_inspect: int = 5) -> None:
```

Comment:
This is the sketchiest change I had to make to get the test suite to run locally. Needs you eyeballs @RobertCraigie